src/Pokemon/AbilityItem.php

```diff
@@ -0,0 +1,57 @@
+<?php
+/**
+ * Created by PhpStorm.
+ * User: zack
+ * Date: 22.12.17
+ * Time: 17:20
+ */
+
+namespace PokeAPI\Pokemon;
+
+use Doctrine\Common\Collections\ArrayCollection;
+
+/**
+ * Class AbilityItem
+ * @package PokeAPI\Pokemon
+ */
+class AbilityItem
```

Comment:
The PokéAPI name of this entity is `PokemonAbility`, maybe we should respect that. `AbilityItem` might lure people into thinking this refers to items bound to abilities.

Reply:
👌